lib/OpenPayU/v2/Refund.php

```diff
@@ -13,28 +13,34 @@ class OpenPayU_Refund extends OpenPayU
 {
     /**
      * Function make refund for order
-     * @param $orderId
-     * @param $description
-     * @param int $amount Amount of refund in pennies
+     * @param $refund
      * @return null|OpenPayU_Result
      * @throws OpenPayU_Exception
      */
-    public static function create($orderId, $description, $amount = null)
+    public static function create($refund)
```

Comment:
This change break API contract.
```suggestion
   public static function create($orderId, $description, $amount = null, $extCustomerId = null, $extRefundId = null)
```

Reply:
sure, I'll change it to optional arguments